test/net/sourceforge/kolmafia/textui/command/GuzzlrCommandTest.java

```diff
@@ -0,0 +1,143 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+import net.sourceforge.kolmafia.AdventureResult;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.KoLConstants;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class GuzzlrCommandTest extends AbstractCommandTestBase {
+  @BeforeEach
+  public void initEach() {
+    KoLCharacter.reset("testUser");
+    KoLCharacter.reset(true);
+    Preferences.resetToDefault("_guzzlrQuestAbandoned");
+    Preferences.resetToDefault("questGuzzlr");
+    Preferences.resetToDefault("guzzlrBronzeDeliveries");
+    Preferences.resetToDefault("guzzlrGoldDeliveries");
+
+    // Stop requests from actually running
+    GenericRequest.sessionId = null;
+  }
+
+  public GuzzlrCommandTest() {
+    this.command = "guzzlr";
+  }
+
+  private static void hasGuzzlrTablet() {
+    AdventureResult.addResultToList(KoLConstants.inventory, ItemPool.get(ItemPool.GUZZLR_TABLET));
```

Comment:
hasGuzzlrTablet sounds like it shout return a boolean.  maybe a more descriptive name?

Reply:
changed to `playerHasGuzzlrTablet`.